contents/updates/content-updates.html

```diff
@@ -0,0 +1,45 @@
+---
+title: Updates
+layout: layout-post
+permalink: /updates/
+---
+
+<h2>Migrating to v2</h2>
+<h6>Last updated on July 14th, 2022</h6>
+<p>
+  Yes, we are finally migrating to a new portal soon! The v2 will be released by
+  the end of this quarter. Tune in here for further updates!
```

Comment:
change this to
We will be upgrading to a new portal soon with a new base framework (details below) which will be released at the end of July. Tune in here for further updates